norddrop/src/device/mod.rs

```diff
@@ -65,7 +65,7 @@ impl NordDropFFI {
                 cb: event_cb,
                 logger,
             }),
-            config: Config::default(),
+            config: Arc::new(Config::default()),
```

Comment:
Probably storing `Arc` here is a huge pain. I would go for storing here as plain data here, and wrap it into arc inside `Service` structure

Reply:
I think it's not that painful and has certain benefits:
- Easy not to think about lifetimes
- Passing the config and updating it in the future makes it just work

Currently, if I will have it as a plain member and wrap it into an Arc<> when passing, I am gonna pass clones instead thus making each transfer with an explicit clone of a config. This is not an issue and depends on the point of view. I changed it into the non-arc member for the piece of mind